### PR TITLE
store preferences in database

### PR DESCRIPTION
**Pull request by JadeSsst**
I've created a function which stores user input from te filter, into the database. I've also changed the object of the schema users so it matches with te filter options.

### CODE REVIEW

File: public/scripts/index.js [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
 const sliders = document.querySelectorAll('.sliders');
 const section = document.querySelector('section');
-const extraFilters = document.querySelector(".extra");
-const basicFilters = document.querySelector(".basics");
+const extraFilters = document.querySelector('.extra');
+const basicFilters = document.querySelector('.basics');
 const titel = document.querySelector('h1');
 
-Array.prototype.forEach.call(sliders,(slider) =>{
+Array.prototype.forEach.call(sliders, (slider) =>{
```
Comment (MarvinMichel) [inline public/scripts/index.js:10]: Why work with the prototype? Isn't this risky?

File: routes/feed.js [hunk 2]
```diff
@@ -23,30 +23,29 @@ router.get('/', async (req, res) => {
...
+  server.Users.findOneAndUpdate(
+    {_id: id},
+    { preferences: {
+      gender: req.body.gender,
+      age: req.body.age,
+      distance: req.body.distance,
+      smoke: req.body.smoke,
+      kids: req.body.kids,
+      hight: req.body.hight }},
+    function(err, result) {
```
Comment (MarvinMichel) [inline routes/feed.js:41]: Try and use an arrow function instead of a ES5 function.

File: public/scripts/index.js (hunk 1 above)
Comment (JadeSsst) [reply public/scripts/index.js]: I made it invisible for now, so we can check at the end if this caused errors

File: routes/feed.js (hunk 2 above)
Comment (JadeSsst) [reply routes/feed.js]: fixed it!

**Comment by IngeDW** [comment]
The code looks good and very consistant. Later on we will have to deside whether to do the client site in English as well, because you have 'Ik zoek...' for example and I use in my labels 'Firstname' etc.

**Pull request merged**